<!DOCTYPE HTML>
<html>
	<head>
		<title>Artigos LBGM</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<link rel="stylesheet" href="https://cdn.datatables.net/2.1.6/css/dataTables.dataTables.min.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header" class="alt">
					<!-- <span class="logo"><img src="images/logo.png" alt="" style="border-radius: 100%; width:21%"></span> -->

						<h1><span class="icon solid fa-folder-open"></span> Artigos Publicados</h1>
						<!-- <p>Ipsum dolor sit amet nullam</p> -->
					</header>

				<!-- Main -->
					<div id="main">
						<!-- Content -->
							<section id="content" class="main">

								<!-- Text -->
									<section>
										<div class="table-wrapper">
											<table id="tableArticles" class="display" style="width:100%">
												<thead>
													<tr>
														<th>Título</th>
														<th>Resumo</th>
														<th></th>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>GO FEAT: a rapid web-based functional annotation tool for genomic and transcriptomic data</td>
														<td style="font-size: x-small;">Downstream analysis of genomic and transcriptomic sequence data is often executed by functional annotation that can be performed by various bioinformatics tools and biological databases. However, a full fast integrated tool is not available for such analysis. Besides, the current available software is not able to produce analytic lists of annotations and graphs to help users in evaluating the output results. Therefore, we present the Gene Ontology Functional Enrichment Annotation Tool (GO FEAT), a free web platform for functional annotation and enrichment of genomic and transcriptomic data based on sequence homology search. The analysis can be customized and visualized as per users’ needs and specifications. GO FEAT is freely and its code is hosted at https://github.com/fabriciopa/gofeat.</td>
														<td style="text-align:center; vertical-align: middle;"><a href="https://dx.doi.org/10.1038/s41598-018-20211-9" target="_blank"><i class="icon solid fa-file-export fa-lg"></i></a></td>
													</tr>
													<tr>
														<td>EasySSR: a user-friendly web application with full command-line features for large-scale batch microsatellite mining and samples comparison</td>
														<td style="font-size: x-small;">Microsatellites, also known as SSRs or STRs, are polymorphic DNA regions with tandem repetitions of a nucleotide motif of size 1–6 base pairs with a broad range of applications in many fields, such as comparative genomics, molecular biology, and forensics. However, the majority of researchers do not have computational training and struggle while running command-line tools or very limited web tools for their SSR research, spending a considerable amount of time learning how to execute the software and conducting the post-processing data tabulation in other tools or manually—time that could be used directly in data analysis. We present EasySSR, a user-friendly web tool with command-line full functionality, designed for practical use in batch identifying and comparing SSRs in sequences, draft, or complete genomes, not requiring previous bioinformatic skills to run. EasySSR requires only a FASTA and an optional GENBANK file of one or more genomes to identify and compare STRs. The tool can automatically analyze and compare SSRs in whole genomes, convert GenBank to PTT files, identify perfect and imperfect SSRs and coding and non-coding regions, compare their frequencies, abundancy, motifs, flanking sequences, and iterations, producing many outputs ready for download such as PTT files, interactive charts, and Excel tables, giving the user the data ready for further analysis in minutes.</td>
														<td style="text-align:center; vertical-align: middle;"><a href="https://doi.org/10.3389/fgene.2023.1228552" target="_blank"><i class="icon solid fa-file-export fa-lg"></i></a></td>
													</tr>
													<tr>
														<td style="vertical-align: middle;">SATIN: a micro and mini satellite mining tool of total genome and coding regions with analysis of perfect repeats polymorphism in coding regions</td>
														<td style="font-size: x-small;"><strong>Background:</strong> 

															Tandem repeats are specific sequences in genomic DNA repeated in tandem that are present in all organisms. Among the subcategories of TRs we have Satellite repeats, that is divided into macrosatellites, minisatellites, and microsatellites, being the last two of specific interest because they can identify polymorphisms between organisms due to their instability. Currently, most mining tools focus on Simple Sequence Repeats (SSR) mining, and only a few can identify SSRs in the coding regions.
															<strong>Results: </strong> 
															
															We developed a microsatellite mining software called SATIN (Micro and Mini SATellite IdentificatioN tool) based on a new sliding window algorithm written in C and Python. It represents a new approach to SSR mining by addressing the limitations of existing tools, particularly in coding region SSR mining. SATIN is available at https://github.com/labgm/SATIN.git. It was shown to be the second fastest for perfect and compound SSR mining. It can identify SSRs from coding regions plus SSRs with motif sizes bigger than 6. Besides the SSR mining, SATIN can also analyze SSRs polymorphism on coding-regions from pre-determined groups, and identify SSRs differentially abundant among them on a per-gene basis. To validate, we analyzed SSRs from two groups of Escherichia coli (K12 and O157) and compared the results with 5 known SSRs from coding regions. SATIN identified all 5 SSRs from 237 genes with at least one SSR on it.
															<strong>Conclusions:</strong>
															
															The SATIN is a novel microsatellite search software that utilizes an innovative sliding window technique based on a numerical list for repeat region search to identify perfect, and composite SSRs while generating comprehensible and analyzable outputs. It is a tool capable of using files in fasta or GenBank format as input for microsatellite mining, also being able to identify SSRs present in coding regions for GenBank files. In conclusion, we expect SATIN to help identify potential SSRs to be used as genetic markers.</td>
														<td style="text-align:center; vertical-align: middle;"><a href="https://doi.org/10.1186/s12859-024-05842-2" target="_blank"><i class="icon solid fa-file-export fa-lg"></i></a></td>
													</tr>
													<tr>
														<td></td>
														<td style="font-size: x-small;"></td>
														<td style="text-align:center; vertical-align: middle;"><a href="" target="_blank"><i class="icon solid fa-file-export fa-lg"></i></a></td>
													</tr>
													<tr>
														<td></td>
														<td style="font-size: x-small;"></td>
														<td style="text-align:center; vertical-align: middle;"><a href="" target="_blank"><i class="icon solid fa-file-export fa-lg"></i></a></td>
													</tr>
													<tr>
														<td></td>
														<td style="font-size: x-small;"></td>
														<td style="text-align:center; vertical-align: middle;"><a href="" target="_blank"><i class="icon solid fa-file-export fa-lg"></i></a></td>
													</tr>
												</tbody>
												<!-- <tfoot>
													<tr>
														<td colspan="2"></td>
														<td>100.00</td>
													</tr>
												</tfoot> -->
											</table>
										</div>			
							</section>
					</div>
				<!-- Footer -->
				<footer id="footer">
					<section>
						<h2>Laboratório de Bioinformática e Genética de Microrganismos - LBGM</h2>
						<!-- <p>Sed lorem ipsum dolor sit amet et nullam consequat feugiat consequat magna adipiscing tempus etiam dolore veroeros. eget dapibus mauris. Cras aliquet, nisl ut viverra sollicitudin, ligula erat egestas velit, vitae tincidunt odio.</p> -->
						<!-- <ul class="actions">
							<li><a href="generic.html" class="button">Learn More</a></li>
						</ul> -->
					</section>
					<section>
						<h2>Onde estamos?</h2>
						<dl class="alt">
							<dt>Endereço</dt>
							<dd>Centro de Computação de Alto Desempenho - CCAD</dd>
							<!-- <dt>Phone</dt> -->
							<!-- <dd>(000) 000-0000 x 0000</dd> -->
							<dt> </dt>
							<dd>Rua Bettina Ferro de Souza - Universitário, Belém - PA, 66075-110</dd>
							<!-- <dd><a href="#">information@untitled.tld</a></dd> -->
						</dl>
						<ul class="icons">
							<!-- <li><a href="#" class="icon brands fa-twitter alt"><span class="label">Twitter</span></a></li> -->
							<li><a href="#" class="icon brands fa-facebook-f alt"><span class="label">Facebook</span></a></li>
							<li><a href="https://www.instagram.com/labbgm_ufpa/" class="icon brands fa-instagram alt"><span class="label">Instagram</span></a></li>
							<li><a href="https://github.com/labgm/" class="icon brands fa-github alt"><span class="label">GitHub</span></a></li>
							<!-- <li><a href="#" class="icon brands fa-dribbble alt"><span class="label">Dribbble</span></a></li> -->
						</ul>
					</section>
					<p class="copyright">LBGM - Laboratory of Genomics and Bioinformatics - UFPA - <span id="date"></span> &copy; All rights reserved.</p>
					<script>
						const d = new Date();
						document.getElementById("date").innerHTML = d.getFullYear();
					</script>
				</footer>

			</div>

		<!-- Scripts -->
		<script src="assets/js/jquery.min.js"></script>
		<script src="assets/js/jquery.scrollex.min.js"></script>
		<script src="assets/js/jquery.scrolly.min.js"></script>
		<script src="assets/js/browser.min.js"></script>
		<script src="assets/js/breakpoints.min.js"></script>
		<script src="assets/js/util.js"></script>
		<script src="assets/js/main.js"></script>
		<script src="https://cdn.datatables.net/2.1.6/js/dataTables.min.js"></script>
		<script>
				new DataTable('#tableArticles', {
					order: [[3, 'desc']]
				});
			</script>
	</body>
</html>